Dfe.Academies.Academisation.Domain/ProjectAggregate/Project.cs

```diff
@@ -41,11 +41,42 @@ public static CreateResult<IProject> Create(IApplication application, Establishm
 
 		var projectDetails = new ProjectDetails(
 			school.Urn,
-			misEstablishmentDetails.Laestab,
-			school.SchoolName
+			misEstablishmentDetails.Laestab
 		)
-		{ UkPrn = establishmentDetails.Ukprn };
+		{ 
+			UkPrn = establishmentDetails.Ukprn,
+			//LocalAuthority = school.LocalAuthority.LocalAuthorityName,
+			//ApplicationReferenceNumber = application.ApplicationId
+			ProjectStatus = "Converter Pre-AO (C)",
+			//ApplicationReceivedDate = application.ApplicationSubmittedOn
+			OpeningDate = DateTime.Today.AddMonths(6),
+			//TrustReferenceNumber = application.ExistingTrust.ReferenceNumber
+			//NameOfTrust = application.ExistingTrust.TrustName
+			AcademyTypeAndRoute = "Converter",
+			ProposedAcademyOpeningDate = school.ConversionTargetDate,
+			ConversionSupportGrantAmount = 25000,
+			PublishedAdmissionNumber = school.CapacityPublishedAdmissionsNumber.ToString(),
+			PartOfPfiScheme = ToYesNoString(school.LandAndBuildings.PartOfPfiScheme),
+			//FinancialDeficit = ToYesNoString(school.SchoolCFYCapitalIsDeficit),
+			//RationaleForTrust = school.SchoolConversionReasonsForJoining,
+			//EqualitiesImpactAssessmentConsidered = ToYesNoString(school.SchoolAdEqualitiesImpactAssessment),
```

Comment:
hi @jordan-beard , 'EqualitiesImpactAssessmentConsidered' has just gone in via my PR.

Trust & anything financial don;t exist yet. Financial next on my list